.yamato/build_linux_x64.sh

```diff
@@ -0,0 +1,14 @@
+sudo apt-get install -y schroot
+sudo apt-get install -y binutils debootstrap
+git submodule update --init --recursive
+git submodule update --init --recursive
```

Comment:
This is here twice.

Reply:
Its failing sometimes due to network issues and retry also fails. So manually trying twice. Only on linux.

Reply:
Then minor, but can we use loop and check exit code a few times?